pkg/auth/providers/local/local_auth_provider.go

```diff
@@ -425,6 +424,18 @@ func indexField(field string, maxindex int) []string {
 	for i := 2; i <= maxindex; i++ {
 		fieldIndexes = append(fieldIndexes, field[0:i])
 	}
+
+	splitToLower := func(s string) []string {
+		var lowers []string
+		for _, v := range strings.Split(s, " ") {
```

Comment:
Should we use `strings.Fields()` instead or do something else to normalize the whitespace?
As it is at the moment indexing "Ta  Da" (note two whitespaces) will produce an empty string index `""`.

Reply:
Great idea, and nicer than strings.Split() and ignoring the spaces.